src/opencl_cryptosafe_fmt_plug.c

```diff
@@ -337,11 +339,9 @@ static int crypt_all(int *pcount, struct db_salt *salt)
 	size_t *lws = local_work_size ? &local_work_size : NULL;
 	size_t gws = GET_NEXT_MULTIPLE(count, local_work_size);
 
-	*pcount *= mask_int_cand.num_int_cand;
+	count *= mask_int_cand.num_int_cand;
```

Comment:
Don't we actually need to update `*pcount`? This change looks like a bug to me.

Reply:
No, `*pcount` isn't used below this. At first, my sanity check (using `count` which didn't account for mask factor) triggered (caught only by TS) and when I looked into that I thought this change was cleaner.

Reply:
Hmm wait, `*pcount` can be used even though we don't return it, right?

Reply:
Yup, fixing that

Reply:
Updated `*pcount` is what tells the caller how many candidates we actually processed (including built-in mask). As to your check, you should either also write the updated value into `count` here or use `*pcount` in the check.